src/Sentry/Internal/BatchProcessorTimer.cs

```diff
@@ -0,0 +1,61 @@
+using System.Timers;
+
+namespace Sentry.Internal;
+
+internal abstract class BatchProcessorTimer : IDisposable
+{
+    protected BatchProcessorTimer()
+    {
+    }
+
+    public abstract bool Enabled { get; set; }
+
+    public abstract event EventHandler<ElapsedEventArgs> Elapsed;
+
+    public void Dispose()
```

Comment:
this is better at the end of the file, so we start reading what really makes this exist isntead of ,NET boilerplate